src/github.com/matrix-org/dendrite/syncapi/sync/notifier.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2017 Vector Creations Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sync
+
+import (
+	"github.com/matrix-org/dendrite/syncapi/types"
+	"github.com/matrix-org/gomatrixserverlib"
+	"sync"
```

Comment:
I've moved `sync` up to the top. I didn't see anything wrong in my editor configuration.

Generally speaking, we don't really focus on formatting when reviewing Go code, because `go fmt`, `goimports`, etc generally do a pretty good job at it. I accept that in this case it seems to have erred since we use `gb`, but in general we just don't think about it and that hasn't been a problem for us. On the contrary, it means we can focus on the code instead.